README.md

```diff
@@ -26,11 +26,23 @@ You can also use the `wishlist` CLI to list and connect to servers in your `~/.s
 #### Remote
 
 If you just want a directory of existing servers, you can use the `wishlist` CLI and a YAML config file. You can also just run it without any arguments to list the servers in your `~/.ssh/config`.
-Check the [example config file](/_example/config.yaml) file as well as `wishlist --help` for details.
+To start wishlist in server mode, you'll need to use the `server` subcommand:
```

Comment:
I think I'd prefer this to be called `serve` (without the 'r').

Reply:
ah, both work actually, but will change the docs!